app/src/main/java/com/duckduckgo/app/email/EmailManager.kt

```diff
@@ -43,16 +42,7 @@ interface EmailManager {
     fun signOut()
     fun getEmailAddress(): String?
     fun getUserData(): String
-    fun waitlistState(): AppEmailManager.WaitlistState
-    fun joinWaitlist(
-        timestamp: Int,
-        token: String
-    )
 
-    fun getInviteCode(): String
-    fun doesCodeAlreadyExist(): Boolean
-    suspend fun fetchInviteCode(): AppEmailManager.FetchCodeResult
-    fun notifyOnJoinedWaitlist()
```

Comment:
I think `FetchCodeResult` and `WaitlistState` sealed classes can be removed as well.

Reply:
Correct! Nice catch!